### PR TITLE
translate-history

### PR DESCRIPTION
**Pull request by silverskyvicto**
Signed-off-by: Yuuki Ebihara <yuuki.ebihara813@gmail.com>

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Thank you for this PR! It helped me realize we needed to update the history page, so I am doing that work in https://github.com/helm/helm-www/pull/767/files (if you'd like to track that PR so you can update this one).

File: content/ja/docs/community/history.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "プロジェクトの歴史"
+description: "プロジェクトの歴史の概要を提供します。"
+aliases: ["/docs/history/"]
```
Comment (bridgetkromhout) [inline content/ja/docs/community/history.md:4]: Please remove this alias - it's a redirection for an old URL.

File: content/ja/docs/community/history.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+weight: 4
+---
+
+Helm 3 は、
+[インキュベーションの最終段階](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc) の
+[CNCF プロジェクト](https://www.cncf.io/projects/) です。
```
Comment (bridgetkromhout) [inline content/ja/docs/community/history.md:10]: Please update this to say "Helm is a [graduated](https://helm.sh/blog/celebrating-helms-cncf-graduation/)
[CNCF project](https://www.cncf.io/projects/)." See https://github.com/helm/helm-www/pull/767/files.

File: content/ja/docs/community/history.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Helm はトップレベルの管理組織を形成し、
+Monocular、Helm Chart Repo、Chart Museum、そして後に Helm Hub など、
+いくつかのプロジェクトが Helm プロジェクトに組み込まれました。
+
+Helm 3 の開発サイクルが始まると、Tiller が削除され、
+Helm はクライアントツールであるという当初のビジョンに近づきました。
+ただし、Helm 3 は引き続き Kubernetes クラスター内のリリースを追跡しているため、
+チームが共同で Helm リリースのセットで作業することができます。
+
+Helm 3 は 2019 年 11 月にリリースされました。
```
Comment (bridgetkromhout) [inline content/ja/docs/community/history.md:33]: Please add "Helm graduated as a CNCF project in April 2020." (See https://github.com/helm/helm-www/pull/767/files)

**Comment by bridgetkromhout** [comment]
Hi, @silverskyvicto! I think this is almost ready to merge. Please make these few small updates and then we can get it merged. Thank you!

**Comment by silverskyvicto** [comment]
Hi @bridgetkromhout.

I fixed some part. Please check my pull request.

**Pull request merged**